commit 9dc6de446d87de65cf7a1174e9bf40f047da88e7
Author: Volker Theile <vtheile@suse.com>
Date:   Tue Mar 12 10:06:01 2019 +0100

    mgr: Cleanup get_localized_module_option function
    
    Address comments from https://github.com/ceph/ceph/pull/26736.
    
    Signed-off-by: Volker Theile <vtheile@suse.com>

diff --git a/src/mgr/ActivePyModules.h b/src/mgr/ActivePyModules.h
index cf6106c5d4a..758f94f0647 100644
--- a/src/mgr/ActivePyModules.h
+++ b/src/mgr/ActivePyModules.h
@@ -116,7 +116,7 @@ public:
 
   PyObject *get_typed_config(const std::string &module_name,
 			     const std::string &key,
-			     const std::string &prefix) const;
+			     const std::string &prefix = "") const;
 
   void set_health_checks(const std::string& module_name,
 			 health_check_map_t&& checks);
